Add camunda modeler 4.0.0-alpha.1 release (#139)
camunda/blog.camunda.org: content/post/2020/03/camunda-modeler-4.0.0-alpha.1-released.md

+++
author = "Oguz Eroglu"
categories = ["Modeling"]
tags = ["BPMN 2.0", "Tooling", "Release Note", "Camunda Modeler"]
date = "2020-03-25T15:00:00+02:00"
title = "Camunda Modeler 4.0.0-alpha.1 Released"
+++

We are happy to announce the release of Camunda Modeler v4.0.0-alpha.1. It is shipped with notable changes in DMN modeling and the Deployment Tool. In addition, we included several bug fixes.

> The most important feature of this release is support for DMN 1.3. Even though DMN 1.3 is not released yet, Camunda Modeler v4.0.0-alpha.1 is ready for it. Note that DMN 1.3 is supported by Camunda BPM 7.13.0 alpha releases, **that is why Camunda Modeler v4.0.0-alpha.1 DMN modeling features should be used with Camunda BPM platform version 7.13.0-alpha.0 and above.**

Reviewer: ```suggestion
This release ships with support for the upcoming DMN 1.3 standard. It can open DMN 1.1 and DMN 1.2 diagrams, however, doing so, it converts these diagrams to DMN 1.3. 

__Ensure you only convert your DMN diagrams if you are using a Camunda BPM platform that supports DMN 1.3, too.__  The upcoming Camunda BPM 7.13.0-alpha3 release as well as patch versions of Camunda Enterprise ship  DMN 1.3 support in the platform. 
```

[Download the latest release](https://downloads.camunda.cloud/release/camunda-modeler/4.0.0-alpha.1/) and start modeling right away.

<!--more-->

> ## Support for latest DMN standards (DMN 1.2 & DMN 1.3)

Reviewer: ```suggestion
## Ready for DMN 1.3
```

> Camunda Modeler v4.0.0-alpha.1 supports opening DMN 1.2 diagrams. Note that as of now, DMN 1.2 is [the latest version of the standard](https://www.omg.org/spec/DMN/). Even though DMN 1.3 is not released yet, Camunda Modeler supports opening such diagrams as well, as version 1.3 will soon become the latest DMN standard. For DMN 1.1 diagrams, Camunda Modeler v4.0.0-alpha.1 automatically migrates them to DMN 1.3 on diagram opening.

Reviewer: ```suggestion
DMN 1.3 is the upcoming version of the DMN standard. Compared to DMN 1.1 version 1.3 of the standard ships with a number of improvements, most importantly a diagram interchange that allows graphical information to be exchanged across vendors. 

The app now exports DMN 1.3 and uses the standardized diagram interchange when doing so. You can open older DMN 1.1 or DMN 1.2 files, however those will be migrated to DMN 1.3 on save.
```

> One additional feature Camunda Modeler v4.0.0-alpha.1 is shipped with is the auto migration to DMN 1.3 on diagram save. It means that Camunda Modeler v4.0.0-alpha.1 automatically saves DMN diagrams compatible with DMN 1.3 standard, independent from the original diagram version.

Reviewer: Drop, if you accept my previous suggestion.
